docs/apis/avalanchego/avalanchego-release-notes.md

```diff
@@ -6,6 +6,20 @@
 
 :::
 
+:::info
+
+If you have an ID for an node, you can use the 
+[avalanche notify service](/docs/nodes/maintain/avalanche-notify.md) 
+to be notified about new releases.
+
+In addition, if you don't have a node ID, you can get release notifications from github.
```

Comment:
```suggestion
In addition, or if you don't have a node ID, you can get release notifications from github.
```

Reply:
Please make the same modification to the other place in this PR.

Reply:
fixed 415e7fb55a826fe9cbc3dc82d45d1e7ad050bbee